libcontainer/cgroups/systemd/v2.go

```diff
@@ -292,6 +296,32 @@ func (m *unifiedManager) Apply(pid int) error {
 	if err := fs2.CreateCgroupPath(m.path, m.cgroups); err != nil {
 		return err
 	}
+
+	goodToChown := []string{"cgroup.procs", "cgroup.subtree_control", "cgroup.threads"}
+	member := func(haystack []string, needle string) bool {
+		for _, v := range haystack {
+			if v == needle {
+				return true
+			}
+		}
+		return false
+	}
+	if m.uid >= 0 {
+		err := filepath.Walk(m.path, func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				return err
+			}
+			if path == m.path || member(goodToChown, info.Name()) {
+				return os.Chown(path, m.uid, -1)
+			} else {
```

Comment:
`else` is redundant here

Reply:
I prefer if / else to be balanced.  Does it violate a formal style guide?

Reply:
Not really, but some linters do complain, and personally I find it redundant, too -- because of `return`.

Feel free to keep though.